client/common/sass/components/_statistics-table.sass

```diff
@@ -20,7 +20,7 @@
 			align-items: center
 
 			+tablet-up
-				grid-template-columns:  436px 100px 30px 1fr
+				grid-template-columns:  clamp(20.00rem, calc(10.66rem + 12.96vw), 27.25rem) clamp(0.00rem, calc(-8.05rem + 11.17vw), 6.25rem) 30px 1fr
```

Comment:
Strongly endorse the use of `clamp` here, this seems like exactly what it's meant for.